docs/user/extract-text.md

```diff
@@ -119,7 +119,7 @@ def visitor_svg_rect(op, args, cm, tm):
 
 
 def visitor_svg_text(text, cm, tm, fontDict, fontSize):
```

Comment:
```suggestion
def visitor_svg_text(text, cm, tm, font_dict, font_size):
```

Although unrelated, we should probably change this to be consistent with the first example (and follow the naming from PEP8).